Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -9,17 +9,14 @@
 import time
 import re
 
```

Comment:
```suggestion
import urllib3
```

Due to an upgrade of MyPy this will fix the build fails

Reply:
Done